examples/.scripts/run-examples.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC3043,SC3020,SC3040,SC3044
+
+# Run each example in parallel and sort the outputs
+
+set -euo pipefail
+
+# Match directories with blobs recursively
+# https://www.gnu.org/software/bash/manual/html_node/The-Shopt-Builtin.html
+shopt -s globstar
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+EXAMPLES_DIR="${SCRIPT_DIR}/.."
+cd "$EXAMPLES_DIR"
+
+export APALACHE_DIST=../quint/_build/apalache
+# Start the apalache server
+"$APALACHE_DIST"/bin/apalache-mc server &> /dev/null &
+
+# We recursively glob all the .qnt files, and run ./run-example.sh on each.
+#
+# Since the output of each run is a row in the README.md table,
+# sorting the outputs lexographically gets the results in the
+# right order.
+parallel "${SCRIPT_DIR}/run-example.sh" ::: **/*.qnt | env LC_ALL=C sort --ignore-case
```

Comment:
Does this really work? We still have global mutable state in Apalache.

Reply:
Do you have in mind where? I have been tearing the server via parallel calls (tho not rigorously) and have generally not had any trouble since I made the parsing threadsafe, as far as I can recall. But I have just started to see signs of worry with the river spec.

Reply:
For context: all the txm tests we currently use for running the `verify` command run concurrently.

Reply:
I was just thinking, in general, of Scala singletons

Reply:
It had been working, but I’m seeing issues with the river spec just added, so maybe the others were working only because of how simple they were.

I’ll need to debug when I have a chance, and maybe we’ll end up needing to sequence this.